Cargo.toml

```diff
@@ -1,14 +1,15 @@
 [package]
 name = "{{project-name}}"
-version = "0.0.1"
+version = "0.1.0"
 authors = ["{{authors}}"]
 edition = "2018"
 
 [lib]
 crate-type = ["cdylib"]
 
 [dependencies]
-wascc-actor = "0.7.1"
+wapc-guest = "0.4.0"
+actor-core = { git = "https://github.com/wascc/actor-interfaces", branch = "main" }
```

Comment:
This should be `https://github.com/wasmcloud/actor-interfaces`. Should we go ahead and add a `version` argument here for the core interface so that it defaults to the `crates.io` dependency when it's released, or should we go ahead and file an issue here to remove the git dependency?

Reply:
let's file an issue to remove the git.. I'll fix the URL.